docs/component.md

```diff
@@ -90,59 +90,62 @@ element by calling `new` on the component’s constructor function:
 
 ```js
 const HelloWorld = Component.extend({
-  tag: "hello-world",
-  view: "Hello <content>world</content> <ul>{{#each(items)}} {{>itemPartial}} {{/each}}</ul>",
-  ViewModel: {
-    items: {
-      default: () => []
-    }
-  }
+	tag: "hello-world",
+	view: `
+		<can-slot name="greetingTemplate" />
+		<content>world</content>
+		<ul>{{#each(items)}} {{this}} {{/each}}</ul>
+	`,
+	ViewModel: {
+		items: {}
+	}
 });
 
 // Create a new instance of our component
 const componentInstance = new HelloWorld({
 
 	// values with which to initialize the component’s view model
 	viewModel: {
-		items: ["eat", "sleep", "code"]
+		items: ["eat"]
 	},
 
 	// can-stache template to replace any <content> elements in the component’s view
 	content: "<em>{{message}}</em>",
 
-	// scope with which to render the <content>
-	scope: {
-		message: "friend"
+	// <can-template> strings rendered by can-stache with the scope
+	templates: {
+		greetingTemplate: "{{greeting}}"
 	},
 
-	// partials made available to the component’s view
-	partials: {
-		itemPartial: "<li>{{this}}</li>"
+	// scope with which to render the <content> and templates
+	scope: {
+		greeting: "Hello",
+		message: "friend"
 	}
 });
 
-myGreetingInstance.element; // is <my-greeting>Hello <em>friend</em> <ul> <li>eat</li>  <li>sleep</li>  <li>code</li> </ul></my-greeting>
+myGreetingInstance.element; // is like <my-greeting>Hello <em>friend</em> <ul> <li>eat</li> </ul></my-greeting>
```

Comment:
why add `like`?

Reply:
The actual template is rendered with newlines, so I wanted to somehow communicate “this isn’t the exact output you’d get.”